Lib/test/test_typing.py

```diff
@@ -1307,7 +1304,7 @@ def test_callable_args_are_correct(self):
         i = Callable[[None], *Ts]
         j = Callable[[None], Unpack[Ts]]
         self.assertEqual(i.__args__, (type(None), *Ts))
-        self.assertEqual(i.__args__, (type(None), Unpack[Ts]))
+        self.assertEqual(j.__args__, (type(None), Unpack[Ts]))
```

Comment:
The `j` variable is currently unused. Given the pattern of the assertions immediately below, it seems pretty clear that it was meant to be used here.